tests/__snapshots__/SettingsMigrationTest__it_migrates_pre_v5.0.0_settings_with_data_set_empty__1.json

```diff
@@ -1,463 +1,468 @@
 {
-  "account": {
-    "id": "account",
```

Comment:
formatting is niet goed, je editor settings kloppen niet. installeer je husky hooks